### DISCUSSION TITLE
Talk-llama on Windows

### DISCUSSION THREAD

**Discussion by VoicelessCoffee** [ggml-org/whisper.cpp#982] [Q&A]
Hello,
Does anybody know how to run Talk-llama on Windows. I tried to use cmake like this:
"cd /examples/talk-llama
cmake -S . -B build/ -D CMAKE_BUILD_TYPE=Release
cmake --build build/ --config Release"

but it does not create anything i can use/execute.

Thanks in advance for any help!

**Comment by DjChick3n** [comment]
Hey just stubbled across your comment while also looking for how to do this. Did you ever find a solution?

**Comment by VoicelessCoffee** [comment]
Nah, sadly not

**Comment by tamo** [comment]
https://github.com/ggerganov/whisper.cpp/tree/master/examples/talk-llama#building
> The talk-llama tool depends on SDL2 library

Download SDL2, `set SDL2_DIR=(the folder you installed SDL2 to)/cmake`, and `cmake -DWHISPER_SDL2=ON` from the root directory. You don't have to enter example/talk-llama.

https://github.com/ggerganov/whisper.cpp/blob/c46886f5991b0119db5eee76ad75e3801b240e7f/.github/workflows/build.yml#L178-L194

**Reply by DjChick3n** [reply]
Thanks, got it up and ruinning.
